control-plane/roles/gardener/tasks/control-plane-networks/gcp.yaml

```diff
@@ -34,7 +34,7 @@
 
 - name: Set pod cidr
   set_fact:
-    _gardener_soil_pod_cidr: "{{ lookup('k8s', kind='Node')[0].get('spec', {}).get('podCIDR') }}"
+    _gardener_soil_pod_cidr: "{{ (gcp_cluster.stdout | from_json)['ipAllocationPolicy']['clusterIpv4CidrBlock'] }}"
```

Comment:
the chances below are also required

Reply:
This deployment role is a bit different from the one used by FITS, so this is not required here.